livedata/src/main/java/io/getstream/chat/android/livedata/controller/ChannelControllerImpl.kt

```diff
@@ -469,6 +469,8 @@ internal class ChannelControllerImpl(
         }
 
         newMessage.user = domainImpl.currentUser
+        // TODO: type should be a sealed/class or enum at the client level
+        newMessage.type = if (newMessage.text.startsWith("/")) { "ephemeral" } else { "regular" }
```

Comment:
Could we extract those hardcoded values out of the code?
Ideally, `type` should be an enum, but at least we should keep them into some constant instead of having hardcoded value around the code

Reply:
we have that in the UI layer already, lets just fix it properly